engine/src/main/scala/cromwell/jobstore/JobStoreReader.scala

```diff
@@ -1,36 +0,0 @@
-package cromwell.jobstore
-
-import akka.actor.{Actor, ActorLogging, Props}
-import akka.event.LoggingReceive
-import cromwell.backend.BackendJobDescriptorKey
-
-object JobStoreReader {
-  def props() = Props(new JobStoreReader())
-}
-
-class JobStoreReader extends Actor with ActorLogging {
```

Comment:
TOL: This isn't a complaint so much as an explanation... I had this as a separate actor so that you could, in theory, create multiple readers without impeding or interrupting the singleton writer